scripts/linters/js_ts_linter.py

```diff
@@ -1014,6 +1014,142 @@ def _check_comments(self):
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+
+                if not (
+                        'private ' + camel_case_class_name + ': ' + class_name
+                        in oppia_angular_root):
+                    total_error_count += 1
+                    summary_message = (
+                        'Please add the class %s to Oppia Angular'
+                        ' Root constructor:' % (class_name)
+                    )
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    summary_message = (
+                        'private %s: %s' % (camel_case_class_name, class_name))
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+
+                if not (
+                        'OppiaAngularRootComponent.' + camel_case_class_name +
+                        ' = ' + 'this.' + camel_case_class_name + ';' in
+                        oppia_angular_root):
+                    total_error_count += 1
+                    summary_message = (
+                        'The static variable hasn\'t been assigned value:')
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    summary_message = (
+                        '  OppiaAngularRootComponent.%s = this.%s' % (
+                            camel_case_class_name, camel_case_class_name))
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+
+                if not '\'' + class_name + '\'' in oppia_root_directive:
+                    total_error_count += 1
+                    summary_message = (
+                        'The class %s hasn\'t been added to ANGULAR_SERVICES'
+                        ' in  oppia-root.directive.ts.' % (
+                            class_name))
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+
+        with linter_utils.redirect_stdout(sys.stdout):
+            if self.verbose_mode_enabled:
+                python_utils.PRINT('----------------------------------------')
+            if total_error_count:
+                python_utils.PRINT('(%s files checked, %s errors found)' % (
+                    total_files_checked, total_error_count))
+                summary_message = linter_utils.FAILED_MESSAGE_PREFIX
+                summary_message += 'OppiaAngularRootComponent linting failed,'
+                summary_message += ' fix the errors listed above'
+                summary_messages.append(summary_message)
```

Comment:
Format these messages, like you have done above.

Reply:
Done!